src/types/spot.ts

```diff
@@ -10,9 +9,9 @@ import {
   OrderTimeInForce,
   OrderType,
   RateLimiter,
-  SymbolFilter,
-  StringBoolean,
   SideEffects,
+  StringBoolean,
```

Comment:
This doesn't seem to be used? Also, nitpick is that it's differently written vs `numberInString` (although I admittedly prefer your version)